sdk-api-src/content/audioenginebaseapo/index.md

```diff
@@ -16,7 +16,8 @@ f1_keywords:
 
 ## -description
 
-This header is part of the Audio Devices DDI Reference. For more information, see:
+This header is used by multiple technologies. For more information, see:
```

Comment:
Some of the paragraphs used to link to the relevant tech tree were inconsistent. For example, this one was `This header is part of the ...` rather than `This header is used by ...`. This change normalizes everything to be consistent on the latter as that was the most prevalent I found.

Reply:
@tannergooding  This is a good change, thanks.